{"summary":{"outcome":"Failed","testsRan":3,"passing":0,"failing":3,"skipped":0,"passRate":"0%","failRate":"100%","skipRate":"0%","testStartTime":"2025-08-05T18:56:16.000Z","testSetupTimeInMs":416,"testExecutionTimeInMs":0,"testTotalTimeInMs":416,"commandTimeInMs":1565,"hostname":"https://orgfarm-d89e3e6fd7-dev-ed.develop.my.salesforce.com","orgId":"00DgL00000713kbUAA","username":"ajayp9402351@agentforce.com","testRunId":"707gL00000B3WhI","userId":"005gL000004tq0UQAQ"},"tests":[{"id":"07MgL0000018lZ4UAI","queueItemId":"709gL0000039a9PQAQ","stackTrace":"Class.LeadCounterTest.setupData: line 17, column 1","message":"System.DmlException: Insert failed. First exception on row 0; first error: DUPLICATE_USERNAME, Duplicate Username.<br>The username already exists in this or another Salesforce organization. Usernames must be unique across all Salesforce organizations. To resolve, use a different username (it doesn't need to match the user's email address). : [Username]","asyncApexJobId":"707gL00000B3WhIQAV","methodName":"TestLeadsCountWhenConverted","outcome":"Fail","apexLogId":null,"apexClass":{"id":"01pgL0000047xVhQAI","name":"LeadCounterTest","namespacePrefix":null,"fullName":"LeadCounterTest"},"runTime":0,"testTimestamp":"2025-08-05T18:56:17.000+0000","fullName":"LeadCounterTest.TestLeadsCountWhenConverted","diagnostic":{"exceptionMessage":"System.DmlException: Insert failed. First exception on row 0; first error: DUPLICATE_USERNAME, Duplicate Username.<br>The username already exists in this or another Salesforce organization. Usernames must be unique across all Salesforce organizations. To resolve, use a different username (it doesn't need to match the user's email address). : [Username]","exceptionStackTrace":"Class.LeadCounterTest.setupData: line 17, column 1","className":"LeadCounterTest","compileProblem":"","lineNumber":17,"columnNumber":1}},{"id":"07MgL0000018lZ5UAI","queueItemId":"709gL0000039a9PQAQ","stackTrace":"Class.LeadCounterTest.setupData: line 17, column 1","message":"System.DmlException: Insert failed. First exception on row 0; first error: DUPLICATE_USERNAME, Duplicate Username.<br>The username already exists in this or another Salesforce organization. Usernames must be unique across all Salesforce organizations. To resolve, use a different username (it doesn't need to match the user's email address). : [Username]","asyncApexJobId":"707gL00000B3WhIQAV","methodName":"TestLeadsCountWhenNotConverted","outcome":"Fail","apexLogId":null,"apexClass":{"id":"01pgL0000047xVhQAI","name":"LeadCounterTest","namespacePrefix":null,"fullName":"LeadCounterTest"},"runTime":0,"testTimestamp":"2025-08-05T18:56:17.000+0000","fullName":"LeadCounterTest.TestLeadsCountWhenNotConverted","diagnostic":{"exceptionMessage":"System.DmlException: Insert failed. First exception on row 0; first error: DUPLICATE_USERNAME, Duplicate Username.<br>The username already exists in this or another Salesforce organization. Usernames must be unique across all Salesforce organizations. To resolve, use a different username (it doesn't need to match the user's email address). : [Username]","exceptionStackTrace":"Class.LeadCounterTest.setupData: line 17, column 1","className":"LeadCounterTest","compileProblem":"","lineNumber":17,"columnNumber":1}}]}